docker/Dockerfile

```diff
@@ -0,0 +1,104 @@
+# This Dockerfile configures a Docker environment that 
+# contains all the required packages for the tool
+FROM ubuntu:22.04
+ARG UID
+ARG GID
+RUN echo "Group ID: $GID"
+RUN echo "User ID: $UID"
+
+USER root
+RUN apt-get update -y && apt-get install apt-utils -y
+RUN DEBIAN_FRONTEND="noninteractive" apt-get -y install tzdata
+
+# Install basic packages 
+RUN apt-get upgrade -y 
+RUN apt-get update -y \
+    && apt-get install -y verilator gcc-riscv64-unknown-elf \
+                          libgmp-dev python3 python3-pip g++\
+                          clang llvm lld clang-tidy clang-format \
+                          gcc-multilib gcc cmake sudo wget vim \
+                          curl tmux git bc
+
+# Install CHERI dependencies
+RUN apt-get update -y \
+    && apt-get install -y autoconf automake libtool pkg-config \
+                          clang bison cmake mercurial ninja-build \
+                          samba flex texinfo time libglib2.0-dev \
+                          libpixman-1-dev libarchive-dev libarchive-tools \
+                          libbz2-dev libattr1-dev libcap-ng-dev \
+                          libexpat1-dev libgmp-dev
+
+# Install bsc build dependencies
+RUN sudo apt-get update && \
```

Comment:
probably unnecessary if we grab a release build rather than building ourselves (maybe some runtime deps are in there though)